components/browser/tabstray/src/main/java/mozilla/components/browser/tabstray/TabViewHolder.kt

```diff
@@ -23,9 +22,6 @@ class TabViewHolder(
     itemView: View,
     private val tabsTray: BrowserTabsTray
 ) : RecyclerView.ViewHolder(itemView) {
-    private val cardView: CardView = (itemView as CardView).apply {
```

Comment:
We can actually keep the elevation here by removing the reference to `cardView` and casting to `CardView`.

```kotlin
itemView.apply {
    elevation = tabsTray.styling.itemElevation
}
```

Reply:
That said that is only if we need the elevation styling.

Reply:
It doesn't appear that we need it; it's having no effect.

Reply:
Marking this as resolve then.

Reply:
I did look at the designs and we have no need for elevation. So, I am a bit more confident here.